plugins/secretstores/all/all.go

```diff
@@ -1 +1,6 @@
 package all
+
+import (
+	// Importing for initialization side effects to register JWTGenerator with SecretStore
+	_ "github.com/influxdata/telegraf/plugins/secretstores/jwt"
+)
```

Comment:
Please create an own file called `plugins/secretstores/all/jwt.go` just as all the other secretstores do.